src/python-common/tox.ini

```diff
@@ -3,9 +3,11 @@ envlist = py27, py3, lint
 skip_missing_interpreters = true
 
 [testenv]
-deps=
-  pytest
-commands=py.test -v {posargs:ceph/tests}
+deps=-rrequirements.txt
+commands=python setup.py test --addopts="--mypy --mypy-ignore-missing-imports" {posargs}
```

Comment:
does this imply python2?

Reply:
There is a special overrride for `[testenv:py27]` below

Reply:
i still don't get it. i think it's
```
[testenv:py27]
commands=python setup.py test {posargs}
```
which overrides the global setting.

so "py3" and "lint" will be using
```
python setup.py test --addopts="--mypy --mypy-ignore-missing-imports" {posargs}
```
for testing.

Reply:
So you would prefer to have a `[testenv:py3]` here? I can do that!

Reply:
done